### PR TITLE
Annotate `utils/uuid.py` and `utils/text.py`

### PR DESCRIPTION
**Pull request by Kludex**
Related to #1511

### CODE REVIEW

File: kombu/utils/text.py [hunk 1]
```diff
@@ -2,7 +2,9 @@
 # flake8: noqa
 
 
+from ctypes import cast
```
Comment (thedrow) [inline kombu/utils/text.py:5]: Do we need to import this?

Comment (Kludex) [reply kombu/utils/text.py:5]: No, it's wrong.

Comment (Kludex) [inline kombu/utils/text.py:5]:
```suggestion
```

Comment (Kludex) [reply kombu/utils/text.py]: Thanks 🙏

**Review by auvipy — CHANGES_REQUESTED** [review]
needs a rebase

**Pull request merged**